app/dashboard/views.py

```diff
@@ -2897,6 +2897,19 @@ def get_profile_tab(request, profile, tab, prev_context):
                     feedbacks__sender_profile=profile
                 ).distinct('pk').nocache()
     elif tab == 'trust':
+        context['is_idena_connected'] = profile.is_idena_connected
+        if profile.is_idena_connected:
+            context['idena_address'] = profile.idena_address
+            context['logout_idena_url'] = reverse(logout_idena, args=(profile.handle,))
+            context['is_idena_verified'] = profile.is_idena_verified()
+            context['idena_status'] = profile.idena_status()
+            if not context['is_idena_verified']:
+                # FIXME: timezone
```

Comment:
what do we need to do to fix this?

Reply:
Please help me to convert the UTC datetime to the user's timezone.